pkg/base1/test-metrics.js

```diff
@@ -127,4 +137,60 @@ QUnit.test("non-instanced decompression", function (assert) {
     assert.deepEqual(sink.samples, [[10], [10]], "got correct samples");
 });
 
+QUnit.test("interval validation", function (assert) {
+    assert.expect(3);
+    const done = assert.async();
+    const metrics_channel = cockpit.channel({
+        payload: "metrics1",
+        source: "internal",
+        interval: -1,
+    });
+
+    metrics_channel.addEventListener("close", (_, error) => {
+        assert.equal(error.problem, "protocol-error");
+        assert.equal(error.message, 'invalid "interval" value: -1');
+        assert.ok(true, "Connection was closed for invalid interval");
```

Comment:
This assertion feels superfluous? We only need these if there's nothing more specific to assert, no?

Reply:
Yeah, can be dropped.